inspirehep/modules/editor/views.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""INSPIRE editor blueprint."""
+
+from __future__ import absolute_import, division, print_function
+
+from flask import (
+    abort,
+    Blueprint,
+    jsonify,
+    request,
+)
+from flask_login import current_user, login_required
+
+from ...utils import tickets
+
+
+blueprint = Blueprint('inspirehep_editor',
+                      __name__,
+                      url_prefix='/editor',)
+
+
+@blueprint.route('/rt/tickets/create', methods=['POST'])
+# TODO: @login_required
+def create_rt_ticket():
+    """View to create an rt ticket"""
+    json = request.json
+    ticket_id = tickets.create_ticket(json['queue'],
```

Comment:
Are you sure `json` will always have all the fields? Otherwise you should protect it with `json.get()`